cmd/minikube/cmd/cache.go

```diff
@@ -67,7 +67,7 @@ var deleteCacheCmd = &cobra.Command{
 }
 
 func imagesInConfigFile() ([]string, error) {
-	configFile, err := config.ReadConfig()
+	configFile, err := config.ReadConfig(constants.ConfigFile)
```

Comment:
we want to make sure, regardless of profile we use the image cache, and not store the images separately. I haven't studied the code but have you verified that it still uses a global image cache and not re-download them per profile? ( could be a good integration test added)

Reply:
noted